server/odc-service/src/main/java/com/oceanbase/odc/service/session/ConnectConsoleService.java

```diff
@@ -589,6 +592,13 @@ private SqlExecuteResult generateResult(@NonNull ConnectionSession connectionSes
         } catch (Exception e) {
             log.warn("Failed to init warning message", e);
         }
+        try {
+            String version = ConnectionSessionUtil.getVersion(connectionSession);
```

Comment:
we may set this flag by checking if there is anything in connection session's cache~

Reply:
Sorry but I thought about it again, this cannot be changed. 
Perhaps this SQL actually supports viewing query profiles, but there is currently no record in the cache. Because the process of caching is asynchronous, we cannot guarantee it.